x-pack/plugins/canvas/public/lib/element.ts

```diff
@@ -25,11 +26,12 @@ export class Element {
   public height?: number;
 
   constructor(config: ElementSpec) {
-    const { name, image, displayName, expression, filter, width, height } = config;
+    const { name, image, displayName, expression, filter, help, width, height } = config;
+    const strings = elementStrings[name];
     this.name = name;
-    this.displayName = displayName || name;
+    this.displayName = (strings && strings.displayName) || displayName || name;
```

Comment:
Why check for the i18n `displayName` and `help` strings here? Since you're wrapping the functions before you register them, you'll already have the translated value here if it exists, or an empty string from the result of `canvas.register`, won't you?

Reply:
This is a merge error, from a previous version of this PR... I'll remove this and update.